servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultHttpSetCookie.java

```diff
@@ -111,18 +119,21 @@ public DefaultHttpSetCookie(final CharSequence name, final CharSequence value,
      * @param secure the <a href="https://tools.ietf.org/html/rfc6265#section-4.1.1">secure-av</a>.
      * @param httpOnly the <a href="https://tools.ietf.org/html/rfc6265#section-4.1.1">httponly-av</a> (see
      * <a href="http://www.owasp.org/index.php/HTTPOnly">HTTP-only</a>).
+     * @param sameSite the
+     * <a href="https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis-05#section-5.3.7">SameSite attribute</a>.
      */
     public DefaultHttpSetCookie(final CharSequence name, final CharSequence value, @Nullable final CharSequence path,
                                 @Nullable final CharSequence domain, @Nullable final CharSequence expires,
-                                @Nullable final Long maxAge, final boolean wrapped, final boolean secure,
-                                final boolean httpOnly) {
+                                @Nullable final Long maxAge, @Nullable final SameSite sameSite, final boolean wrapped,
+                                final boolean secure, final boolean httpOnly) {
```

Comment:
not really related to this PR but I wondered if you may want to consider adding a Builder at some point as these are quite a few params for a constructor.

Reply:
lets consider for a followup PR.